build.sh

```diff
@@ -135,7 +135,7 @@ if [ $# == 1 ] ; then
     BUILD_FE=1
     BUILD_BE=1
     BUILD_BROKER=1
-    BUILD_META_TOOL=1
```

Comment:
why change value from 1 to on here?

Reply:
> why change value from 1 to on here?

to apapt cmakelist
```
if (BUILD_META_TOOL AND BUILD_META_TOOL STREQUAL "ON")
    add_subdirectory(${SRC_DIR}/tools)
```

Reply:
I think it is better to keep one style for build option，how about change cmakelist by modfiy “on” to ”1“？